codegen/generators/server/generator.go

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
I have no idea why this file comes up as a new file, probably a remaining file after a merge/rebase from v2 branch. Is this file safe to be removed from this PR ?